iXBRLViewerPlugin/viewer/src/js/theme.js

```diff
@@ -21,16 +33,17 @@ export function getTheme() {
 }
 
 export function initializeTheme() {
-    const initialTheme = window.matchMedia('(prefers-color-scheme: dark)').matches
-            ? 'dark'
-            : 'light';
-    setTheme(initialTheme);
+    const storedTheme = getStoredTheme();
+    if (storedTheme !== null) {
+        setTheme(storedTheme);
+    } else {
+        setTheme(window.matchMedia(`(prefers-color-scheme: ${DARK_THEME})`).matches ? DARK_THEME : LIGHT_THEME);
```

Comment:
Out of curiosity, why persist the default rather than only when the user makes an explicit choice?

Reply:
Sorry, I misread `setTheme` as `storeTheme`.